fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyAdapter.java

```diff
@@ -226,4 +226,12 @@ public void createForeignKeyConstraint(String constraintName, String schemaName,
         // the tenanteColumnName
         return columns;
     }
+
+    @Override
+    public void createFhirSchemas(String schemaName, String adminSchemaName) {
```

Comment:
We don't technically need this in derby.  I do like having this for consistency.

Reply:
yes, just for consistency.